### PR TITLE
Add Pages publishing to root-signing-staging

### PR DESCRIPTION
**Pull request by jku**
This requires support for "workflow" build type (see https://github.com/sigstore/github-sync/pull/122)

### CODE REVIEW

File: github-sync/github-data/sigstore/repositories.yaml
Comment (bobcallaway) [inline github-sync/github-data/sigstore/repositories.yaml:1463]: This doesn't exist as an option under https://github.com/sigstore/github-sync/blob/201b23090cb3cbc628dc256d683d02342792a629/pkg/config/config.go#L59 - is your use of pages different than our other repos that use it?

Comment (jku) [reply github-sync/github-data/sigstore/repositories.yaml:1463]: yes it is different, see the github-sync link in the PR message.

I maybe shouldn't have marked this "ready for review", sorry (but with two PRs keeping this a draft until the other is merged does likely mean a multi-day review cycle) .

Comment (bobcallaway) [reply github-sync/github-data/sigstore/repositories.yaml:1463]: ah, my bad - i missed that. approved and merged that PR.

**Comment by bobcallaway** [comment]
hmm, this shows no pending changes, i guess this is to be expected?

**Comment by jku** [comment]
> hmm, this shows no pending changes, i guess this is to be expected?

It is weird... Because I used to have this configured manually in root-signing-staging but the previous "pulumi up" from this repo undid my manual changes. So I did expect this to do changes.

**Comment by jku** [comment]
> hmm, this shows no pending changes, i guess this is to be expected?

Actually, I think my github-sync patch is missing the line that applies the changes.

Marking draft for now, thanks for double checking

**Comment by jku** [comment]
Ok, preview looks pretty good now.

(the custom role removal is just because this branch is not based on tip of current tree and is missing #398). That looks totally correct to me but I can rebase if requested.

**Review by cpanato — APPROVED** [review]

**Review by bobcallaway — APPROVED** [review]

**Comment by jku** [comment]
the actual run is still unsuccessful because of the custom role not working: #400 will unblock

**Pull request merged**
